pkg/util/configutils_test.go

```diff
@@ -77,3 +77,75 @@ func TestIsTruthy(t *testing.T) {
 	assert.False(util.IsTruthy("NO"))
 	assert.False(util.IsTruthy("0"))
 }
+
+func TestIsPrometheusServiceMonitor(t *testing.T) {
```

Comment:
Seems like overkill to me considering that we have the Golden tests which include testing the parser, but :shrug:

Reply:
I would like to keep this, it helps if you don't have to run the whole golden test when you are currently parsing a new config value.